### PR TITLE
enh(php) highlight variables

### LINKED ISSUE #2778: PHP no variable highlighting
**Describe the issue**
PHP variables are not highlighted. Most of my code snippets are quite unreadable because of that.

**Which language seems to have the issue?**
PHP

**Expected behavior**
(ignore the background)
image

**Actual behavior**
image

**Additional context**
I can live without semicolons, but variables in PHP are critical.

### PR DESCRIPTION
**Pull request by taufik-nurrohman**
Resolve #2778

### Checklist
- [x] Tested.
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

File: src/languages/php.js [hunk 1]
```diff
@@ -11,38 +11,38 @@ Category: common
  * @returns {LanguageDetail}
  * */
 export default function(hljs) {
-  var VARIABLE = {
+  let VARIABLE = {
```
Comment (joshgoebel) [inline src/languages/php.js:14]: Nice, but most (all?) of these should instead be const. :-)

**Comment by joshgoebel** [comment]
You'll also need to update the tests in `markup/php` to match your grammar change.

**Review by joshgoebel — CHANGES_REQUESTED** [review]
Please update according to notes.  Thanks!

File: src/languages/php.js (hunk 1 above)
Comment (taufik-nurrohman) [reply src/languages/php.js:14]: Okay.

**Comment by joshgoebel** [comment]
Please finish checklist!  Looking good!

**Review by joshgoebel — APPROVED** [review]

**Comment by joshgoebel** [comment]
@taufik-nurrohman Thanks so much!

**Pull request merged**